package.json

```diff
@@ -1150,7 +1150,7 @@
         "@types/node": "^16.18.36",
         "@types/ps-tree": "^1.1.0",
         "@types/semver": "^5.5.0",
-        "@types/vscode": "1.79.0",
+        "@types/vscode": "^1.81.0",
```

Comment:
This is the latest types version. Not sure why 1.82.0 doesn't exist.

Reply:
I've seen that before where the `@types/vscode` package lags behind VSCode itself